util/CodeableConcepts.js

```diff
@@ -0,0 +1,56 @@
+// Codeable Concepts, for now, is a utility library that understand the
+// individual CCs and returns either React components, objects or strings
+// for their values.
+import { compact, get } from 'lodash';
+
+const v20131 = { // English only! There's a German version too!
+  C: 'Emergency Contact',
+  E: 'Employer',
+  F: 'Federal Agency',
+  I: 'Insurance Company',
+  N: 'Next-of-Kin',
+  S: 'State Agency',
+  U: 'Unknown',
+};
+
+export function Relationship(relationship) {
+  if (relationship === undefined || relationship.length === 0) {
+    return '';
+  }
+
+  if (relationship[0].text !== undefined) {
+    return relationship.text;
```

Comment:
if `relationship[0].text !== undefined` then `relationship.text` will definitely be `undefined` right?

Reply:
It could be tighter. This ONLY checks if it's `undefined` and not empty.

Reply:
relationship.text isn't in line with the fhir spec. I'll update this.